app/views/groups/index.html.slim

```diff
@@ -0,0 +1,3 @@
+div class='container-box d-flex'
+  div class='cards-index ml-5 mr-1 my-3' style='width: 40%'
+  div id='map'style='width: 60%; height: 100vh; overflow: hidden;' data-markers='= @markers.to_json' data-mapbox-api-key='pk.eyJ1IjoiYmxhbWI4ODgiLCJhIjoiY2ttbjdreDB5MG16azJwbXo4dzRzM2lnMyJ9.qae37hyOWjEGEfbYk2mJkw'
```

Comment:
are the markers appearing?

Reply:
Haven't tested but it should. Need some seeds.